packages/extension-base/src/background/handlers/Tabs.ts

```diff
@@ -19,13 +19,17 @@ import RequestExtrinsicSign from '../RequestExtrinsicSign';
 import State from './State';
 import { createSubscription, unsubscribe } from './subscriptions';
 
-function transformAccounts (accounts: SubjectInfo): InjectedAccount[] {
+function transformAccounts (accounts: SubjectInfo, anyType = false): InjectedAccount[] {
   return Object
     .values(accounts)
     .filter(({ json: { meta: { isHidden } } }) => !isHidden)
+    .filter(({ type }) => anyType ? true : type && ['ed25519', 'sr25519', 'ecdsa'].includes(type))
```

Comment:
kept it as a separate filter to make it easier to read, can also integrate it above.

Reply:
Am I right here btw, `ecdsa` isn't SS58, but the interface is compatible is it?

Reply:
Should be fine... hopefully :)